DependencyInjection/App/Presenters/AnotherPresenter.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
+
+namespace App\Presenters;
+
+use App;
+use Vendor;
+
+class AnotherPresenter extends Vendor\Presenters\SomePresenter
+{
+	/**
+	 * @var \Vendor\Services\AnotherVendorService
```

Comment:
klidně, ale takto to generuje phpstorm i ve chvíli, kdy už NS naimportovány - máš to nějak nastavený nebo to otrocky umazáváš?